searchJob/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+from . import views
+
+app_name = 'searchJob'
+
+urlpatterns = [
+    url(r'^$', views.jobs),
```

Comment:
did you check this? I think you're missing an attribute  url(r'^$', views.jobs, name=...)

Reply:
yes,  i checked that and it work well. 
i saw django documentation and i will fix it right now 
thank you

Reply:
yes, i check that and it work well.